app/models/space.rb

```diff
@@ -7,6 +7,7 @@ class Space < ApplicationRecord
   has_many :facility_reviews, dependent: :restrict_with_exception
   has_many :aggregated_facility_reviews, dependent: :restrict_with_exception
   has_many :reviews, dependent: :restrict_with_exception
+  has_many :space_contacts, dependent: :restrict_with_exception
```

Comment:
why not :destroy, but :restrict_with_exception? 

Same goes for all the above ones. A space contact, review, or facility review isn't worth much if the space is deleted?

Reply:
What does restrict_with_exception do?

Reply:
It is like destroy, but also gives us an exception. Makes it easier to debug.